templates/web/pages/bmviii_home_temp.tmpl

```diff
@@ -0,0 +1,265 @@
+    </div>
+    <div class="container bmviii-front-logo" style="top: 70%; padding-left: 5%">
+        <img alt="Purdue University" src={{ static_path "bmviii_purdue_text.svg" }} width="75%">
+    </div>
+</section>
+
+<section class="section has-background-image" id="why" style='background-color: #FF8E9E; background-image: url({{ static_path "bmviii_why_bm_background.svg" }}); padding-bottom: 125px; padding-left: 0; padding-right:0;'>
+	<div class="container" style="padding: 0 1.5rem;">
+		<div class="columns is-centered content">
+			<div class="column is-2 bmviii-section-header" style="margin-left: 48px; margin-right: 12px; margin-top:72px">
+				<img alt="Why BoilerMake?" src={{ static_path "bmviii_why_bm_text.svg" }}>
+			</div>
+		</div>
+	</div>
+	<div class="container" style="display: flex; justify-content: flex-end; padding-top: 48px; margin-right: 0px;">
+	    <div class="column is-full">
+            <div class="box content" style="width:1218px;">
+                <article>
+                    <div class="media-content">
+                        <div class="content">
+                            <p style="font-family: Cabin; family-style: normal; font-weight: 600; font-size: 18px">
+                                In response to the COVID-19 pandemic, and in accordance with university and local guidelines, BoilerMake VIII will be fully virtual.
+                                While the online format will be a new experience for BoilerMake, our dedication to creating an empowering experience for our hackers to develop their projects remains.
+                                Our team has been hard at work conducting our virtual transition to ensure that both our hackers and sponsors have the necessary platforms and tools to build community and undergo an unforgettable and meaningful experience.
+                                If you have any questions, refer to our FAQ or email us at team@boilermake.org for additional information.
+                            </p>
+                        </div>
+                    </div>
+                </article>
+            </div>
+        </div>
+	</div>
+	<div class="container" style="display: flex; justify-content: flex-end; padding-top: 72px;">
+        <div class="column is-half">
+            <p style="font-family: Cabin; family-style: normal; font-size: 28px; color: #000000;">BoilerMake is Purdue University's largest hackathon. Over the past seven years,
+             thousands of students from across the country have come to BoilerMake. This event is for experienced hackers, newbies to technology, and everyone in between.</p>
+        </div>
+    </div>
+    <div class="container" style="display: flex; justify-content: space-between;">
+        <div class="column is-half">
+            <img alt="Blue van" src={{ static_path "bmviii_van.svg" }}>
+        </div>
+        <div class="column is-half"; style="display: flex; justify-content: space-between; padding-top: 120px;">
```

Comment:
The design spec for these looked a little different:
![Screen Shot 2020-10-05 at 21 20 22](https://user-images.githubusercontent.com/10781907/95148137-3305f400-0748-11eb-8196-2c07f072e0df.png)

Reply:
Besides the rings, I updated the van and the Reasons a little more